physionet-django/physionet/urls.py

```diff
@@ -74,7 +74,7 @@
     ),
 ]
 
-if ProjectFiles().is_lightwave_supported:
```

Comment:
Now that I think about it... maybe we can get rid of the `ENABLE_LIGHTWAVE` environment variable completely and just base it off `ProjectFiles().is_lightwave_supported`, i.e. change:
```python
if ENABLE_LIGHTWAVE:
    INSTALLED_APPS += ['lightwave']
```
to
```python
if ProjectFiles().is_lightwave_supported:
    INSTALLED_APPS = ['lightwave']
```

Makes sense semantically. What do you think @rafalkosla101?

Reply:
Seems like it would make a cycle of imports inside app. I think we should be sticking to current implementation.

Reply:
Yep - this is due to us using the settings. Let's keep it the way it is